admin/database/postgres/postgres.go

```diff
@@ -528,6 +539,15 @@ func (c *connection) DeleteUsergroupMember(ctx context.Context, groupID, userID
 	return checkDeleteRow("usergroup member", res, err)
 }
 
+func (c *connection) FindExpiredUserAuthTokens(ctx context.Context) ([]*database.UserAuthToken, error) {
+	var res []*database.UserAuthToken
+	err := c.getDB(ctx).SelectContext(ctx, &res, "SELECT t.* FROM user_auth_tokens t WHERE t.expires_on < now()")
+	if err != nil {
+		return nil, parseErr("auth tokens", err)
+	}
+	return res, nil
+}
```

Comment:
1. You can just change this to `DeleteExpiredUserAuthTokens`
2. Even though it's not a problem, put `WHERE t.expires_on IS NOT NULL AND t.texpires_on < now()` for safety

Reply:
No, I meant to do the deletes directly inside of it as well – you can delete many rows in one statement in SQL :) See `DeleteExpiredDeviceAuthCodes` for an example